### ISSUE TITLE
[idea] Fix accessibility violation "heading-order" at .eStUD.sc-crHjuH:nth-child(1) > .sc-hKwAaq.bAEomI > .sc-eCIkhf.kXwCRe > h5

### ISSUE THREAD

**Issue by nhunsaker** [metatoy/sorb-demo#2]
### Intent

Fix accessibility violation "heading-order" at .eStUD.sc-crHjuH:nth-child(1) > .sc-hKwAaq.bAEomI > .sc-eCIkhf.kXwCRe > h5

### Success criteria

Re-running the a11y audit against https://demo.sorbcloud.com/ no longer reports "heading-order" at .eStUD.sc-crHjuH:nth-child(1) > .sc-hKwAaq.bAEomI > .sc-eCIkhf.kXwCRe > h5

### Altitude

ticket

### Source

brainstorm:a11y-20260816-015841
